src/app/plan/duck/actions.ts

```diff
@@ -192,6 +191,15 @@ const initMigration = (planName: string) => ({
   planName,
 });
 
+const planReconcilePolling = (planValues) => ({
```

Comment:
For now,  @eriknelson and I committed to using the generations field to determine if reconcile is complete. There is some great stuff in this PR but i think we should open a smaller PR to incrementally introduce some of these enhancements.